### DISCUSSION TITLE
Error when downloading on youtube

### DISCUSSION THREAD

**Discussion by jayday565** [aclap-dev/video-downloadhelper#1066] [Help and Q&A]
### Your question

I have a license for Firefox. When I install the add on and enter my license everything works fine. I am able to download videos from youtube.
But after a while videos on youtube still get detected but I always get an error when trying to download 
"Download failed
Sadly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error (it's anonymous!)."
For every video.
It used to work again after reinstalling the add on but not anymore.

### Addon & CoApp version and details

version: 9.0.2.12
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:128.0) Gecko/20100101 Firefox/128.0

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

When you create a problem, you really should click the `Copy information to clipboard` button in the VDH settings & paste all of that into your problem description.  Other bits of useful information are included there that you have omitted.

Have you tried the VDH beta?  Look in the Wiki for that.  A link to the Wiki appears at the top of every page of this forum.  I have been using the beta for a while now & I have not had problems on YouTube.  After you install the beta, if you still have problems on YouTube, post some URLs of problematic YouTube videos.  That way, others can try to replicate your results.

**Comment by jayday565** [comment]
All videos are problematic I can't download any (this on for instance https://www.youtube.com/watch?v=-NN9w4_FtSk). And I tried reinstalling the stable version again it seems to temporarily solve the problem.
I already used the inapp bug report several times but didn't hear back.
I'll give the beta a try if this happens again (probably).

**Reply by GrampaWildWilly** [reply]
Ironic choice of source material.  It's all obsolete now, of course.

My setup is Windows 11 Pro 64-bit version 23H2 build 22631.3958, Firefox 128.0.3 64-bit, licensed VDH 9.0.2.14a1 beta, CoApp 2.0.19.  With that, here's how things went.

![#01](https://github.com/user-attachments/assets/cfd91fd4-68eb-45c0-b3e0-afea6544117a)
![#02](https://github.com/user-attachments/assets/158c9df4-8bed-457e-bdad-d2d826e43fc2)
![#03](https://github.com/user-attachments/assets/934207d4-ab1d-4e53-81c8-20dea6c05234)
![#04](https://github.com/user-attachments/assets/e6d75db7-41ea-4d99-a85e-8f0b799001fe)
![#05](https://github.com/user-attachments/assets/6f6df2b1-f176-42e5-81e4-5ecd6857f63b)
![#06](https://github.com/user-attachments/assets/b07beede-684d-411a-aa93-4a5f5e66a3ce)

So the beta is definitely my recommendation.

Other things like antivirus & firewall can interfere.  There's lots of discussion of that elsewhere in this forum.

Also, there is a long-standing problem with downloading from YouTube when you are logged in with a paying account.  I have a free account & it's never interfered with VDH.  (You can see I'm not logged in during my test case above.  Note the exhortation to `Sign in` in the upper right corner of the YouTube web page.)  But paying accounts seem to present a problem for VDH that has not yet been solved.  The answer in such cases is obviously to sign out.  Folks have observed problems with age-restricted content on YouTube because it requires you to sign in, allegedly to verify your age.  The content is free, it just requires a login.  But your sample doesn't fall into that category.  I don't know why you're having trouble with it.  I'm not.
